packages/kit/src/routes/Tab/Navigator.tsx

```
@@ -12,13 +12,13 @@ import {
   useMedia,
 } from '@onekeyhq/components';
 import platformEnv from '@onekeyhq/shared/src/platformEnv';
+import type { ETabRoutes } from '@onekeyhq/shared/src/routes';
```

ESLint

[error] Type import "ETabRoutes" is used by decorator metadata.

(@typescript-eslint/consistent-type-imports)